packages/flutter_tools/lib/src/convert.dart

```diff
@@ -40,8 +43,9 @@ class Utf8Decoder extends cnv.Utf8Decoder {
     if (reportErrors && result.contains('\u{FFFD}')) {
       throwToolExit(
         'Bad UTF-8 encoding found while decoding string: $result. '
-        'The Flutter team would greatly appreciate if you could file a bug or leave a '
-        'comment on the issue https://github.com/flutter/flutter/issues/15646.\n'
+        'The Flutter team would greatly appreciate if you could file a bug explaining '
+        'exactly what you were doing when this happened:\n'
+        'https://github.com/flutter/flutter/issues/new/choose\n'
```

Comment:
We don't have a tool specific template, but https://github.com/flutter/flutter/issues/new?assignees=&labels=&template=BUG.md&title= might be a better link

Reply:
I considered that but I thought it would make the error message sufficiently uglier as to outweigh the benefit.